src/Controller/Component/CalendarComponent.php

```diff
@@ -64,10 +64,11 @@ public function startup(Event $event) {
 	 * @param string $year Year
 	 * @param string $month Month
 	 * @param int $span Years in both directions
+	 * @param array $options To be passed to previous and next methods
 	 * @throws \Cake\Http\Exception\NotFoundException
 	 * @return void
 	 */
-	public function init($year, $month, $span = 10) {
+	public function init($year, $month, $span = 10, $options = []) {
```

Comment:
```suggestion
	public function init($year, $month, $span = 10, array $options = []) {
```

Reply:
Fixed in 177cdfceb6ae4b3047d913ab957e86ba11109a40